tests/models/reportArticles.test.js

```diff
@@ -0,0 +1,28 @@
+import { expect } from 'chai';
```

Comment:
@theghostyced why is there a change to this file?

Reply:
Ok. Yeah, eslint was flagging errors in this file

Reply:
Okay 👍